Update manage-dynamic-distribution-groups.md (#2644)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-dynamic-distribution-groups/manage-dynamic-distribution-groups.md

> [!IMPORTANT]

> > A dynamic distribution group includes any recipient in Active Directory with attribute values that match its filter. If a recipient's properties are modified to match the filter, the recipient could inadvertently become a group member and start receiving messages that are sent to the group. Well-defined, consistent account provisioning processes will reduce the chances of this issue occurring.
> 
> > [!IMPORTANT]
> > Dynamic distribution groups are not synced from Exchange Online to Azure Active Directory or to your onpremises Active Directory. Therefore features such as Azure Conditional Access does not support being scoped to an Exchange Online dynamic distribution group.

Reviewer: ```suggestion
> - A dynamic distribution group includes any recipient in Active Directory with attribute values that match its filter. If a recipient's properties are modified to match the filter, the recipient could inadvertently become a group member and start receiving messages that are sent to the group. Well-defined, consistent account provisioning processes will reduce the chances of this issue occurring.
> - Dynamic distribution groups are not synced from Exchange Online to Azure Active Directory or to your on-premises Active Directory. Therefore, features such as Azure Conditional Access do not support being scoped to an Exchange Online dynamic distribution group.
```
